### PR TITLE
Rotation pacman

### PR DESCRIPTION
**Pull request by Laura4lilavati**

### CODE REVIEW

File: src/main/java/pacman/ActionKeyPacman.java [hunk 1]
```diff
@@ -9,21 +9,22 @@ public ActionKeyPacman(DrawComponents drawComponents) {
     }
 
     @Override
-    public void keyPressed(KeyEvent event) {}
+    public void keyPressed(KeyEvent event){
```
Comment (julia-jpg) [inline src/main/java/pacman/ActionKeyPacman.java]: please use space before the " {"

File: src/main/java/pacman/ActionKeyPacman.java [hunk 2]
```diff
@@ -1,30 +1,69 @@
...
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.Timer;
 
 public class ActionKeyPacman implements KeyListener {
-    private DrawComponents drawComponents;
+    
+    private final DrawComponents drawComponents;
+    private int keyRight, keyLeft, keyUp, keyDown;
+    Timer timer = new Timer(200, new ActionListener(){
```
Comment (julia-jpg) [inline src/main/java/pacman/ActionKeyPacman.java]: please use space before the " {"

File: src/main/java/pacman/DrawComponents.java [hunk 3]
```diff
@@ -61,26 +62,30 @@ public void moveRight() {
         x = pacman.right();
         y = y;
         repaint();
+        direction="pacmanRight.gif";
```
Comment (maryangela) [inline src/main/java/pacman/DrawComponents.java]: add spaces before and after of "="

File: src/main/java/pacman/DrawComponents.java [hunk 4]
```diff
@@ -61,26 +62,30 @@ public void moveRight() {
...
     }
 
     //Method to move pacman to left 10 pixels
     public void moveLeft() {
         x = pacman.left();
         y = y;
         repaint();
+        direction="pacmanLeft.gif";
```
Comment (eli7890) [inline src/main/java/pacman/DrawComponents.java]: Please use one space before  and after "=".

File: src/main/java/pacman/DrawComponents.java [hunk 5]
```diff
@@ -61,26 +62,30 @@ public void moveRight() {
...
     }
 
     //Method to move pacman to down 10 pixels
     public void moveDown() {
         y = pacman.down();
         x = x;
         repaint();
+        direction="pacmanDown.gif";
```
Comment (eli7890) [inline src/main/java/pacman/DrawComponents.java]: Please use one space before and after "=".

**Pull request merged**